core/kover/dataset/split.py

```diff
@@ -48,13 +48,17 @@ def normal_raise(exception):
 
     # Validate that the genome identifiers refer to genomes in the dataset
     not_in_ds = []
+    train_ids = (open(train_ids_file, 'r').read()).split('\n')
+    train_ids = [i for i in train_ids if i]
```

Comment:
Use `[i.strip() for i in train_ids if i]` to be sure you don't have junk stuff before/after the id.